src/main/java/com/recurly/v3/requests/InvoiceRefund.java

```diff
@@ -73,7 +73,7 @@
    * The amount to be refunded. The amount will be split between the line items. If no amount is
    * specified, it will default to refunding the total refundable amount on the invoice.
    */
-  public Integer getAmount() {
+  public Float getAmount() {
```

Comment:
Let's be sure to call this out in the release notes. And that it's being published as a patch because it's a bugfix.